### ISSUE TITLE
Migrate S_B and S_W to covEst

### ISSUE THREAD

**Issue by gabrielodom** [BenBarnard/classReduce#9]
These functions are covariance estimation functions, so they belong in covEst

**Comment by BenBarnard** [comment]
Their functionality should be expanded to handle lists, data frames, matrices, resample, and grouped df objects.

**Comment by BenBarnard** [comment]
Yes, they belong in `covEstR`, but the current package is further along in development and I don't see holding this package back while we wait for `covEstR`. I think the covariance functions should handle the bare minimum, and then we will update with a `covEstR` import later.
